lib/chef/data_collector/messages.rb

```diff
@@ -68,6 +68,7 @@ def self.run_end_message(reporter_data)
           "id"                     => run_status.run_id,
           "message_version"        => "1.0.0",
           "message_type"           => "run_converge",
+          "node"                   => run_status.node,
```

Comment:
you should probably make sure -- explicitly with a spec test -- that the node.run_context and node.run_context.resource_collection are not getting rendered into the json (and that you're actually hitting node.to_json/for_json correctly).

i'm reasonably certain that its behaving correctly, but it would be really horrible if that broke at some point in the future (possibly through some bugginess in JSON rendering).